src/utils/helpers.js

```diff
@@ -0,0 +1,25 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    btn.textContent = "Saving...";
+  } else {
+    btn.textContent = "Save";
+  }
```

Comment:
Please, reuse `defaultText` and `loadingText`, to be able to redefine the texts if needed

```js

export function setButtonText(
  btn,
  isLoading,
  defaultText = "Save",
  loadingText = "Saving..."
) {
  if (isLoading) {
    btn.textContent = loadingText;
  } else {
    btn.textContent = defaultText;
  }
}
```